scripts/release-binary.sh

```diff
@@ -143,7 +150,7 @@ do
     //tools/docker:envoy_distroless \
     //tools/docker:envoy_ubuntu
 
-  if [ -n "${DST}" -a -n "${PUSH_DOCKER_IMAGE}" ]; then
+  if [ -n "${DST}" -a "${PUSH_DOCKER_IMAGE}" -eq 1 ]; then
```

Comment:
This is now leftover from original code, you can drop `-n "${DST}"` since it's completely unrelated to pushing Docker images.